keyboards/keychron/q1/rev_0100/rules.mk

```diff
@@ -18,4 +18,4 @@ RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 DIP_SWITCH_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
-RGB_MATRIX_DRIVER = IS31FL3733
+RGB_MATRIX_DRIVER = IS31FL3733
```

Comment:
there are still changes in this file

Reply:
The only changes made was removing the "LTO_ENABLE" rule and moving it to the the keymap "rules.mk". I did check again to make sure that nothing else was affected besides that one line.